### ISSUE TITLE
FreeCAD not working with Windows 11

### ISSUE THREAD

**Issue by wieslawzuber** [FreeCAD/FreeCAD#10517]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

After installing and starting FreeCAD, it shows the following error - The application cannot be started because MSVCP140_1.dll is missing

Po zainstalowaniu i uruchomieniu FreeCAD pokazuje taki błąd - Nie można uruchomić aplikacji bo brakuje MSVCP140_1.dll

### Full version info

```shell
The application cannot be started because MSVCP140_1.dll is missing
```

### Subproject(s) affected?

None

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by chennes** [comment]
Did you install 0.21.0, or 0.21.1? In either case the problem can be resolved by installing this package from Microsoft:
https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2015-2017-2019-and-2022

**Comment by wieslawzuber** [comment]
Thx ! Its working 😀

niedz., 3 wrz 2023 o 16:45 Chris Hennes ***@***.***>
napisał(a):

**Comment by chennes** [comment]
Good: which version did you download?

**Comment by wieslawzuber** [comment]
0.21.0

niedz., 3 wrz 2023 o 19:42 Chris Hennes ***@***.***>
napisał(a):

**Comment by chennes** [comment]
Thanks: I believe I fixed this problem with the 0.21.1 installer.

**Comment by Roy-043** [comment]
The mentioned file is indeed present in the 0.21.1 installer. Closing this.
